src/baklava/options/Vector.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="d-sm-flex">
+    <div class="ml-1">{{ name }}</div>
+    <IntegerInc :index=index
+                :key="index"
+                :value=val
+                @value-change="updateValue"
+                v-for="(val, index) in value"></IntegerInc>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import IntegerInc from '@/baklava/input/IntegerInc.vue';
+
+@Component({
+  components: { IntegerInc },
+})
+export default class Vector extends Vue {
+  @Prop() value!: [number];
+
+  @Prop({ type: String }) name!: string;
+
+  updateValue(value: number, index: number) {
+    const copy = [...this.value];
```

Comment:
How come we need to copy the array?

Reply:
You're not allowed to directly modify this.value, because that's up to the Baklava to do, so Vue will complain.